pkg/clusters/addons/kong/kong.go

```diff
@@ -181,6 +223,16 @@ func (a *Addon) Delete(ctx context.Context, cluster clusters.Cluster) error {
 		return fmt.Errorf("%s: %w", stderr.String(), err)
 	}
 
+	if a.license != "" && a.license == KongLicenseSecretName {
+		stderr := new(bytes.Buffer)
+		cmd = exec.Command("kubectl", "delete", "secret", a.license, "--namespace", a.namespace) //nolint:gosec
+		cmd.Stdout = io.Discard
+		cmd.Stderr = stderr
+		if err := cmd.Run(); err != nil {
+			return fmt.Errorf("delete secret err msg %s: %w", stderr.String(), err)
+		}
+	}
+
```

Comment:
This relates to some previous comments: I think we should be generating the license secret in all cases, and just taking the license contents value. Then we can just always clean up the license we deploy.